packages/suite/src/components/wallet/CoinBalance/index.tsx

```diff
@@ -4,7 +4,9 @@ import { Account } from '@wallet-types';
 import { formatCoinBalance } from '@wallet-utils/balanceUtils';
 import { HiddenPlaceholder } from '@suite-components';
 
-const Wrapper = styled.div``;
+const Wrapper = styled.div`
+    padding-right: 10px;
```

Comment:
shouldn't we also add negative margin like before? otherwise it'll affect the scenario in which another component is placed next to it (eg. in dashboard https://github.com/trezor/trezor-suite/blob/287269484aba3929c713d985379ca1eaa987c0b2/packages/suite/src/views/dashboard/components/AssetsCard/components/Asset/index.tsx#L133)

Reply:
That's true. 👍 I even found a better place where to put the hack so we'll be safer regarding cutoffs.